insights/util/subproc.py

```diff
@@ -57,15 +62,19 @@ def __init__(self, *cmds, **kwargs):
     def _build_pipes(self, out_stream=PIPE):
         log.debug("Executing: %s" % str(self.cmds))
```

Comment:
Completely unrelated, but I just noticed this while reading the surrounding code. Not for this PR, but a general question.

Why isn't this using the interpolation that `logging` already has built in? That way you don't pay the `str()` and interpolation overhead if the logging level is above debug.
```suggestion
        log.debug("Executing: %s", self.cmds)
```

Reply:
Based on your suggestion, I've un-done the previous line breaks in the new commit to keep the changes minimal for this update. Thanks again @ekohl